### PR TITLE
Blog user personas questionnaire results

### PR DESCRIPTION
**Pull request by yns000**

### CODE REVIEW

**Comment by kingthorin** [comment]
To address the DCO requirement you'll need to sign-off the commit(s):

- https://github.com/zaproxy/zaproxy/blob/main/CONTRIBUTING.md#developer-certificate-of-origin
- https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff

File: site/content/blog/2023-12-18-user-personas-poll/index.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+- blog
+- questionnaire 
+date: "2023-12-20"
+authors:
+- yiannis
+---
+
+The Zed Attack Proxy (ZAP) team is dedicated to making our tool the best it can be for our diverse user base. As the cybersecurity landscape evolves, so do the needs of our users. That's why we're excited to announce the **ZAP User Personas Questionnaire** — an initiative to better understand and cater to the unique requirements of everyone who uses ZAP.
+
+### **Why This Matters**
```
Comment (kingthorin) [inline site/content/blog/2023-12-18-user-personas-poll/index.md:18]: Headigs shouldn't also need bold tags. (Throughout)

**Review by kingthorin — COMMENTED** [review]
I'll do a full read/review over the weekend.

File: site/content/blog/2023-12-18-user-personas-poll/index.md (hunk 1 above)
Comment (thc202) [reply site/content/blog/2023-12-18-user-personas-poll/index.md:18]: This is an old blog post, added by accident? https://www.zaproxy.org/blog/2023-12-21-user-personas-poll/

**Comment by psiinon** [comment]
Looks good :)
Some general comments:

- Worth adding an intro? linking to https://www.zaproxy.org/blog/2023-12-21-user-personas-poll/
- Include some charts! I can help if you have problems with them...
- Remove the poll link from the front page? (I assume we're closing it now?)
- At some point would be good to have a new (non blog) page detailing the types of users we know about

**Review by psiinon — APPROVED** [review]

**Review by thc202 — APPROVED** [review]

**Pull request merged**